### ISSUE TITLE
Add a way to search for assets

### ISSUE THREAD

**Issue by rcorre** [itchio/itch.io#1177]
Currently you can search https://itch.io/game-assets by tag, but if you enter a search term into the top search bar, it takes you to https://itch.io/search which searches everything (games, assets, ect.). Is it possible to implement searching by an arbitrary search term for _only_ assets? Thanks!

**Comment by albertvaka** [comment]
Big +1 here! Right now one can only search by tags and not by name. I was looking for "sewer" pixel art backgrounds, and this one didn't show up because it isn't tagged as "sewer": https://lil-cthulhu.itch.io/pixel-art-sewer-background I only found it using Google.

**Comment by pixelpax** [comment]
yes please! SOooo much quality content made unusable by people who need it not being able to find it :(

**Comment by DMClVG** [comment]
Annoyed by this as well. +10000

**Comment by KvartzCode** [comment]
Agreed. This needs to be implemented!

**Comment by tilleul** [comment]
Not to be an ass but it's quite incredible that this feature has been request for 5+ years (according to google searches) and it's still not here !

What's the unreasonable reason to NOT implement this ?

**Comment by InfiniteProductions** [comment]
Totally agree with you folks.

> What's the unreasonable reason to NOT implement this ?

There is not, it looks like these past years, companies running well known websites hold with code made by people who know how to work, hiring only intern level morons who are there to try to maintain it in operational condition but unable to add the easiest feature. This when they are actually able to run it properly, cause Bricklink for instance is a very unbelievable case, they still shutdown completely the website for 15 mins a day and a full hour a month for backup & maintenance purpose ! It's not a joke, and they still collect fees from sellers who use their website to sell Lego parts & sets.

I witness this for many many websites including some famous ones, looks at selling websites like Amazon, Ebay, many specialized shops (sport, toys, whatever), there are all flawed, full of bad items, not in stock items, discounts/offers issues, you name it.
It's a big shame actually but it is what it is unfortunately.

I guess the only way to put pressure is to not use the website for a full day, a full week or more as a punishment, with complete ZERO activity for so much time, they will be force to do something, or they are just as great as governments who are in deep denial.

**Comment by ErnSur** [comment]
I'm also losing my mind over it. Tag filtering should be an addition to text search, not a replacement.

**Comment by KoBeWi** [comment]
Actually there *is* a way to search for assets, but I can't believe this is so dumb.
I tried to bypass the lack of search by using Google, like `in:itch.io/game-assets bones` and it took me to a page `itch.io/game-assets/tag-bones`, which is a page of non-existent tag, but lists the assets I looked for just fine.
Which means that
tag search
is just keyword search
limited to an arbitrary set of words
🤦‍♂️

The specific keyword I wanted is not a valid tag. You can't type it in the "select a tag" field. Pressing Enter will just do nothing. But you can just go to `itch.io/game-assets/tag-<anything you can think of>` and it will display it as if it was a valid tag.

It makes no sense, but at least it works I guess 🤷‍♂️

**Comment by laniidae** [comment]
Actually, multi-tag searching seems to work, use it like `itch.io/game-assets/tag-icons/tag-weapons`.

**Comment by awardell** [comment]
I can see why they wouldn't want to just turn that into a search bar with no modification. The problem is that it looks for EXPLICIT string matches. For example,

https://itch.io/game-assets/tag-grave/tag-tileset
vs
https://itch.io/game-assets/tag-graves/tag-tileset
vs
https://itch.io/game-assets/tag-graveyard/tag-tileset
vs
https://itch.io/game-assets/tag-cemetery/tag-tileset
etc.

It has no ability to provide results with synonyms or associated words. Having that exposed through a search bar could give users the impression that there isn't whatever they're looking for, thus hiding valid related assets.

**Comment by servel333** [comment]
https://itch.io/t/202023/search-assets

6 years ago. ☠️
